system_tests/bigquery.py

```diff
@@ -117,7 +117,6 @@ def test_create_table(self):
         self.to_delete.insert(0, table)
         self.assertTrue(table.exists())
         self.assertEqual(table.name, TABLE_NAME)
-        self.assertTrue(table._dataset is dataset)
```

Comment:
-1 for just dropping the assertion:  perhaps we'd be better off exposing a read-only `dataset` property?